### PR TITLE
VIH-6905 - Show the "share screen" button for anyone who isn't a witness

### PR DESCRIPTION
**Pull request by tremolo64**
### JIRA link (if applicable) ###
VIH-6905

### Change description ###
Show the "share screen" button for anyone who isn't an observer

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.ts [hunk 1]
```diff
@@ -36,26 +37,23 @@ export abstract class HearingControlsBaseComponent implements OnInit, OnDestroy
...
         this.displayConfirmPopup = false;
     }
 
     get canShowScreenShareButton(): boolean {
-        const allowedRoles = [Role.Representative, Role.JudicialOfficeHolder, Role.Judge];
-        if (allowedRoles.includes(this.participant.role)) {
-            return true;
-        }
+        const isNotTablet = !this.deviceTypeService.isTablet();
+        const isAllowedRole = this.participant?.hearing_role !== HearingRole.WITNESS;
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.ts:54]: the ticket requests VIH-6905
As a: Hearing Participant (Judge, JoH, Litigant in Person, Representative, Witness)
This looks like everyone but a witness can screen share.
If you want to stop all observers, that's a hearing role in it self

**Review by shaed-parkar — CHANGES_REQUESTED** [review]
Double-check the hearing role filter when determining who can see the screen share button

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.ts (hunk 1 above)
Comment (tremolo64) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.ts:54]: The ticket changed AC - now, anyone who isn't a witness is allowed to screen share

**Review by shaed-parkar — APPROVED** [review]
Nice work!

**Pull request merged**